src/pages/AddJob.jsx

```diff
@@ -12,8 +12,6 @@ import jobValidationSchema from '../data/jobValidationSchema'
 import { useNavigate } from 'react-router-dom'
 import { IoArrowBack } from 'react-icons/io5'
 import { UserAuth } from '../auth/AuthContext'
-import { addJob } from '../utils/jobHistoryOptimized'
-import React, { useState, useEffect, useActionState } from 'react'

```

Comment:
deleting this line breaks the page:
```suggestion
import React, { useState, useEffect, useActionState } from 'react'
```